src/UITests/LoadEventFile.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using System.Reflection;
+
+namespace UITests
+{
+    /// <summary>
+    /// Start the app, open an A11yEvent file, spot check loaded data, exit app
+    /// </summary>
+    [TestClass]
+    public class LoadEventFile : AIWinSession
+    {
+        static readonly string EventFileName = "WildlifeManagerTest.a11yevent";
+        static readonly string TestFilePath = $"{Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)}\\TestFiles";
+
+        /// <summary>
+        /// The entry point for this test scenario. Every TestMethod  will restart ai-win, so
+        /// we want to use them sparingly.
+        /// </summary>
+        [TestMethod]
+        [TestCategory("NoStrongName")]
+        [TestCategory("UITest")]
+        public void LoadEventFileTests()
+        {
+            driver.Maximize();
+            CheckEventLog();
+            CheckEventDetails();
+            CheckPropertyView();
+            CheckPatternsView();
+            ScanWindow();
+        }
+
+        private void ScanWindow()
+        {
+            var issueCount = driver.ScanAIWin(TestContext);
+            Assert.AreEqual(0, issueCount);
+        }
+
+        /// <summary>
+        /// Click the first row in the event log
+        /// </summary>
+        private void CheckEventLog()
+        {
+            var eventGrid = driver.FindElementByAccessibilityId("dgEvents");
+            var rows = eventGrid.FindElementsByClassName("DataGridRow");
+            Assert.AreEqual(rows.Count, 10);
+
+            rows[0].Click();
+
+            // Text is populated after the cell above is selected (blank otherwise)
+            Assert.AreEqual("09:58:37.859, EventRecorderNotification, Event Recorder", rows[0].Text);
+        }
+
+        /// <summary>
+        /// Validate the event details pane in the bottom right
+        /// </summary>
+        private void CheckEventDetails()
+        {
+            var detailGrid = driver.FindElementByAccessibilityId("dgEventDetails");
+            var rows = detailGrid.FindElementsByClassName("DataGridRow");
+
+            // rows.Count is 0, need to understand why
```

Comment:
@jalkire let's chat about these when you're back in

Reply:
Maybe you already found this, but if you scope to the entire event mode control (AutomationId=ctrlEventMode), you find these 3 rows as well as the other 10. If you scope to the left side (AutomationId=ctrlEvents), you find the other 10. If you scope to the right side (AutomationId=ctrlTabs), you find 0. So the 3 rows we care about here disappear as soon as you scope to a parent below the entire event mode. If we can't figure out why this is happening, we can at least use that higher scoping to verify the count is 10 + 3...

Reply:
Using the totals suggestion